subprojects/docs/src/docs/userguide/javaPlugin.adoc

```diff
@@ -628,36 +628,80 @@ Starting with Gradle 1.10, it is possible to include only specific tests, based
 
 Test filtering feature has following characteristic:
 
-* Fully qualified class name or fully qualified method name is supported, e.g. `org.gradle.SomeTest`, `org.gradle.SomeTest.someMethod`
-* Wildcard '*' is supported for matching any characters
+* Fully qualified class name or fully qualified method name is supported, e.g. `org.gradle.SomeTest`, `org.gradle.SomeTest.someMethod`.
+* Simple class name or simple method name is supported if the pattern starts with an upper-case letter, e.g. `SomeTest`, `SomeTest.someMethod` (Since 4.7).
+* Wildcard '*' is supported for matching any characters, however, abusing wildcards might lead to performance problems - see instructions below.
```

Comment:
I don't think we need to call this out anymore, since we now support the simple name matching. This removes the reason why people were putting a wild card at the front.

Reply:
You mean we totally remove this feature or just remove them in documentation?

What if `--tests '*IntegrationTest'` to run all integration tests? I think we should keep this pattern.

Reply:
You can still do that, but I don't think anyone expects this to be super fast.

Reply:
Understand. So let's just remove the performance part.